README.md

```diff
@@ -206,3 +206,17 @@ To use Postman for testing the PostgREST API corrrectly accessing the database:
 ### Adding new configuration properties
 To add a new configuration property to the database, simply add an entry for it in `appconfig_properties.csv` (under ./dataFiles). Then, simply refresh the database using the `refresh_database.sh` script.
 
+## MkDocs Set up with docker
+
+Follow the following steps:
+
+1. Open a terminal & make sure docker is running
+2. cd documentation
+3. run docker pull polinux/mkdocs
```

Comment:
Can we maybe add all the commands to run in bash script so it's easier to see, copy and run?

Reply:
Added!